elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -174,6 +179,15 @@ def lookup_embedding(self, unique_ids):
         embedding_vectors = np.concatenate(embedding_vectors, axis=0)
         return embedding_vectors.reshape((len(keys), self.output_dim))
 
+    def _check_id_valid(self, ids):
+        if self.input_dim is not None and np.sum(ids >= self.input_dim) > 0:
```

Comment:
We can try more efficient way instead of np.sum(ids >= self.input_dim) and ids[ids >= self.input_dim][0]

Reply:
Done,  use np.argmax(ids >= self.input_dim).